.github/workflows/test.yml

```diff
@@ -122,12 +122,17 @@ jobs:
         containers: [1, 2, 3, 4]
     steps:
       - uses: actions/checkout@v3
+
+      # Sets ownership, which is misconfigured when checking out a new container.
+      # See https://github.com/actions/runner/issues/2033#issuecomment-1204205989.
+      - run: chown -R $(id -u):$(id -g) $PWD
+
       - uses: ./.github/actions/setup
       - uses: actions/cache@v3
         id: cypress-cache
         with:
           path: /root/.cache/Cypress
-          key: ${{ runner.os }}-cypress
+          key: ${{ runner.os }}-cypress-12-12-0
```

Comment:
what does this suffix mean?

Reply:
This reflects the cypress version, 12.12.0.
It could be generated programmatically, something like `yarn list cypress --silent` or hashfiles of `node_modules/cypress`.

Reply:
updated to use `**/node_modules/cypress/package.json`.